### ISSUE TITLE
original Error not available in custom error formatter

### ISSUE THREAD

**Issue by francescosalvi** [nestjs/graphql#2723]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

Since we upgraded to the latest package version, it seems like the `originalError` is not present anymore when hitting `ApolloServerOptionsBase.formatError`.

As per Apollo documentation, https://www.apollographql.com/docs/apollo-server/data/errors/#for-client-responses
> The formatError hook receives two arguments: the first is the error formatted as a JSON object (to be sent with the response), and the second is the original error (wrapped in GraphQLError if thrown by a resolver).

The second part of the statement is apparently no longer true (see the two not-even-`Error` `Objects` in the image):
![image](https://user-images.githubusercontent.com/17246673/225949701-4a219bc1-0131-442d-82f8-948b67f46c4b.png)

### Minimum reproduction code

none, very trivial to repro

### Steps to reproduce

just throw any custom Error from within a resolver callback

### Expected behavior

`formatError` should have access to the original custom Error

### Package version

11.0.3

### Graphql version

`graphql`: 16.6.0
`@apollo/server`: 4.5.0

### NestJS version

9.3.10

### Node.js version

16.14.2

### In which operating systems have you tested?

- [ ] macOS
- [ ] Windows
- [X] Linux

### Other

I can solve the issue two ways:
* setting `autoTransformHttpErrors: false` in `GraphQLModule` options (this bypasses `wrapFormatErrorFn` entirely)
* editing `ApolloBaseDriver.createTransformHttpErrorFn` like so:
  ```
  if (!isHttpException) {
  +++ return originalError;
  ---  return formattedError;
  }
  ```
  Basically reverting this
  
![image](https://user-images.githubusercontent.com/17246673/225948135-1d292a86-9d3e-4ba5-820f-3758f1d0bd16.png)

**Comment by Kasheftin** [comment]
Is there any chance it's broken again? I'm using @nestjs/graphql v13.1.0, and the originalError is not in error.extensions. The other project used v12.1.1 and it worked fine.

**Comment by NilsMoller** [comment]
@Kasheftin The original error seems to be in `error.OriginalError`
